lib/common/graph.js

```diff
@@ -204,6 +204,24 @@ class ConcertoGraphVisitor extends DiagramVisitor {
         parameters.graph.addVertex(scalarDeclaration.getFullyQualifiedName());
     }
 
+    /**
+     * Visitor design pattern
+     * @param {Decorator} decorator - the object being visited
+     * @param {Object} parameters  - the parameter
+     * @protected
+     */
+    visitDecorator(decorator, parameters) {
+        const parent = decorator.getParent();
+        const modelFile = parent.getModelFile();
+        decorator.getArguments()?.forEach(argument => {
+            const fqn = modelFile.getFullyQualifiedTypeName(argument.name);
+            if (fqn) {
+                parameters.graph.addVertex(fqn);
+                parameters.graph.addEdge(fqn, parent.getFullyQualifiedName());
```

Comment:
I think that the edge should be in the other direction, no?

Reply:
Really? I would expect that the decorator assumes the role of a superType.

Reply:
Just rethinking this now, and I could translate it to mean that the declaration imports the decorator and that would invert the edge direction.